### ISSUE TITLE
PyBEL namespace resources cause errors in processing corpora

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#779]
One way to reproduce this issue is 
```python
from indra.sources import bel
bel.process_belscript('indra/data/large_corpus.bel')
```
which errors with
```python
~/Dropbox/postdoc/darpa/src/indra/indra/sources/bel/processor.py in get_mapping(self, key, value)
    568         resource = self.resources.get(key)
    569         if resource is not None:
--> 570             return resource['Values'][value]
    571 
    572 

KeyError: 'peroxisome'
```
Looking deeper, the `resource['Values']` dict, which is loaded via `pybel` looks like this:
```
{'Acrosome': 'D000177', ... 'Peroxisomes': 'D020675', ... 'trans-Golgi Network': 'D021601'}
```
and indeed it does not include `peroxisome` as a key. 

One explanation is that the namespace resource itself changed, i.e. it used to contain `peroxisome` but it doesn't anymore. Another explanation is that the way context is handled in PyBEL (or the parameterization of the API call in INDRA towards PyBEL) changed and context markers that were previously ignored are now attempted to be processed, causing errors. Perhaps, previously, the default behavior eliminated these keys in PyBEL hence they were never encountered in INDRA? If so, is there an argument we can pass to PyBEL to restore this behavior?

**Comment by bgyori** [comment]
@cthoyt Any help or clarification would be great!

**Comment by cthoyt** [comment]
I’m super bogged down with finishing the icon proposal this week. I promise I’ll get to writing some tests and figuring it out first thing next week :)

**Comment by cthoyt** [comment]
I did a bit of sleuthing and from my perspective, it looks like these errors are occurring because the source BEL document did not properly name terms using the given namespaces. This occurred a lot because the curators did not pay attention when they were using the Cell Line Ontology vs MeSH, or basically any ontology versus MeSH equivalents.

Sorry that's not so helpful - I think it's because I am only now properly passing the flag to the namespace:name parser and before, these statements were getting thrown away during PyBEL compilation and not making it to INDRA conversion.

New PR coming up in a minute with some better logging for this. As of now, I found only 42 instances in the small corpus where this was an issue, so it's surely not systemic.
